src/ticketmap/index.tsx

```diff
@@ -327,39 +339,49 @@ export default class TicketMap extends Component<Props, State> {
    * Public Methods
    */
 
-  highlightSection = (section: string) => this.toggleSectionHighlight(section, true)
+  highlightSection = (section: string): void => this.toggleSectionHighlight(section, true)
 
-  unhighlightSection = (section: string) => this.toggleSectionHighlight(section, false)
+  unhighlightSection = (section?: string): void => {
+    if (!section) {
+      this.setState({ currentHoveredSection: null })
+      return this.updateMap()
+    }
+    return this.toggleSectionHighlight(section, false)
+  }
 
-  toggleSectionHighlight = (section: string, shouldHighlight = true) => {
+  toggleSectionHighlight = (section?: string, shouldHighlight: boolean = true): void => {
```

Comment:
Why are you hinting that this function returns `void` when it contains a `return` statement?

Reply:
It contains multiple return statements, are you talking specifically about `return this.fillSection(sectionId, shouldHighlight)`? We could remove that final return, I personally like making it clear when I expect functions to return.

Reply:
Also, you indicated that you are requesting changes made before this PR could merge, what are you requesting be changed?

Reply:
Actually having pondered this question overnight I realize my answer should've been this: There are three points in the function in which it could exit, at each point it returns `undefined`, therefore I typed the return as `void` in the method signature.